build.sbt

```diff
@@ -161,6 +161,16 @@ lazy val core = crossProject(JSPlatform, JVMPlatform).in(file("core"))
   .settings(dottyLibrarySettings)
   .settings(dottyJsSettings(ThisBuild / crossScalaVersions))
 
+/**
+ * Implementations of concurrent data structures (Ref, MVar, etc) purely in
+ * terms of cats effect typeclasses (no dependency on IO)
+ */
+lazy val concurrent = crossProject(JSPlatform, JVMPlatform).in(file("concurrent"))
+  .dependsOn(kernel)
+  .settings(name := "cats-effect-concurrent")
+  .settings(dottyLibrarySettings)
```

Comment:
Random note: you don't need the `dottyLibrarySettings` bit here, since you don't have any dependencies!

Reply:
Cool, thanks!